src/main/java/org/killbill/billing/plugin/adyen/client/model/paymentinfo/KlarnaPaymentInfo.java

```diff
@@ -177,6 +190,27 @@ public String getAdditionalData() {
         return additionalData;
     }
 
+    private void hashIdentifiers() {
+        // hash sensitive identifiers
+        if (!identifierHashed) {
+            for (Seller seller : sellers) {
+                final String merchantId = seller.getMerchantId();
+                final String hashedId = UUID.nameUUIDFromBytes(merchantId.getBytes()).toString();
```

Comment:
Based on discussion with OneTrust team (privacy review) it was decided we will send the hashed id for merchantId and accountId (customerId) since these identifiers are used internally in Groupon to reference customer/merchant data.